public/app/assetlibrary/assetLibraryFactory.js

```diff
@@ -71,9 +71,6 @@
       page = page || 0;
       searchOptions = searchOptions || {};
 
-      // Certain types of sort will narrow the search
-      utilService.narrowSearchPerSort(searchOptions);
-
```

Comment:
Narrowing the search cannot be the default because whiteboardsReuseController, as an example, needs all assets sorted by my_pinned.